shared/common-adapters/animation.desktop.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+import * as React from 'react'
+import Box from './box'
+import animationData from './animation-data.json'
+import type {Props} from './animation'
+
+const defaultDimension = 16
+
+class Animation extends React.Component<Props> {
+  render() {
+    // jest doesnt' support canvas out of the box, so lets just not do anything
+    if (__STORYSHOT__) {
+      return (
+        <Box>
+          {JSON.stringify({
+            height: this.props.height || defaultDimension,
+            options: {animationData: animationData[this.props.animationType]},
+            style: this.props.style,
+            width: this.props.width || defaultDimension,
+          })}
+        </Box>
+      )
+    }
+    // jest pukes if the import is on top so just defer till render
+    const Lottie = require('lottie-react-web').default
```

Comment:
on import this thing queries canvas so if that doesn't exist (like in storyshots) then it'll die so we just move it down here. it is possible to make a fake canvas for jest but its really not worth it for us